README.md

```diff
@@ -122,6 +130,12 @@ I (272) cpu_start: Starting scheduler on PRO CPU.
 I (0) cpu_start: Starting scheduler on APP CPU.
 Hello, world!
 ```
+##### Backtrace 
```

Comment:
If we open the topic of backtrace decoding, shouldn't we mention that decoding for riscv chips is incorrect or maybe even wrong by using this approach? And point the reader to the EH frame setup?

Reply:
i could add a footnote, and add a little section on what to add tot he bottom?

Reply:
Anything would work as long as it is clear for the users that what is suggested in this paragraph won't work well on anything but xtensa...

Reply:
Or else we would be opening even more statements of "it does not work as per the documentation"